src/components/AuthenticationPages/auth.css

```diff
@@ -0,0 +1,108 @@
+.main {
+	background-color: transparent !important;
+	margin-top: 5rem !important;
```

Comment:
try not to use `!important`. Instead of that, you can increase the specificity and get your work done.